frontend/src/pages/pipelines/global/modelCustomization/startRunModal/StartRunModal.tsx

```diff
@@ -47,10 +50,17 @@ const StartRunModal: React.FC<StartRunModalProps> = ({
         <DashboardModalFooter
           onSubmit={() => {
             if (selectedProject) {
+              fireFormTrackingEvent(eventName, { outcome: TrackingOutcome.submit });
               onSubmit(selectedProject);
```

Comment:
This modal submission does not start the Lab Tune. It prompts for a project to use, validates that project, and then submitting it takes the user to the model customization form where they still need to fill out more details before starting the actual tune (`frontend/src/pages/pipelines/global/modelCustomization/ModelCustomizationForm.tsx`). The cancel event here still probably makes sense, but we probably want to exclude the submit event here or perhaps split this into two events and have this one be something like "Redirected to Lab Tune form from registry".

Reply:
Yes, you are right, that should be a better event name.